content/scalability/docs/cluster-services.md

```diff
@@ -0,0 +1,60 @@
+# Cluster Services
+
+Cluster services run inside an EKS cluster, but they are not user workloads. If you have a Linux server you often need to run services like NTP, syslog, and a container runtime to support your workloads. Cluster services are similar, supporting services that help you automate and operate your cluster. In Kubernetes these are usually run in the kube-system namespace and some are run as DaemonSets.
```

Comment:
The first time a key word, e.g. DaemonSets, is referenced, can you add a hyperlink?

Reply:
My only concern with that is the guide isn't intended to be read from start to finish. Readers should be able to jump to any section at any time and key words are hard to track/link in every document. I've done it for a few things (eg Karpenter) but now it feels like there're too many links for a repeated word.